src/pages/generate/pages/result/curationSheet/CurationSheet.tsx

```
@@ -96,37 +130,37 @@ export const CurationSheet = ({ groupId = null }: CurationSheetProps) => {
         ? byProductId
         : index + 1;
 
-      const originalPrice = Number(product.furnitureProductOriginalPrice);
-      const discountPrice = Number(product.furnitureProductDiscountPrice);
-      const discountRate = Number(product.furnitureProductDiscountRate);
-      const saveCount = Number(product.furnitureProductSaveCount);
+      const originalPrice = toFiniteNumber(product.furnitureProductOriginalPrice);
```

ESLint

[error] Replace `product.furnitureProductOriginalPrice` with `⏎········product.furnitureProductOriginalPrice⏎······`

(prettier/prettier)